CMakeLists.txt

```diff
@@ -33,9 +33,9 @@ set(SUPPORTED_CXX_STD_VERSION "11")
 
 option(ENABLE_LATEST_CXX_STD "Upgrade C++ standard version to ${LATEST_KNOWN_CXX_STD_VERSION}. Required to get rid of unused variables warnings in compilers not supporting [[gnu::*]] attributes. Can bring other benefits, including performance and efficiency ones. Before a pull request build with this disabled." OFF)
 if(ENABLE_LATEST_CXX_STD)
-	set(CMAKE_CXX_STANDARD "${LATEST_KNOWN_CXX_STD_VERSION}")
+  set(CMAKE_CXX_STANDARD "${LATEST_KNOWN_CXX_STD_VERSION}")
```

Comment:
if that's a white space fix (also next 2 diffs), that's independent of TBB device, should be separate pull with minor fixes (not sure if already done)

Reply:
Unfortunately, I did not include these minor things in my separate pull request, because I thought I could handle things differently in files I'm already touching. I now moved this things in a separate commit.